### ISSUE TITLE
compute_stats terminated with an error

### ISSUE THREAD

**Issue by JunTian-CIRA** [PaulHancock/Robbie#21]
I ran Robbie with the command: nextflow -C 2m.config run main.nf -profile common,zeus -resume, but got the following error:

![Screenshot from 2020-02-28 13-22-02](https://user-images.githubusercontent.com/53510317/75512759-74555900-5a2d-11ea-9d3a-ec66a677f524.png)

**Comment by JunTian-CIRA** [comment]
The error is caused by line 106 in auto_corr.py:
"idx = np.random.choice(range(max(NPIX,len(tab))), NPIX, replace=False)"
"idx" can be as large as "NPIX=500", so the "fluxes = tab[idx][flux_cols]" exceeds the length of the table.

**Comment by PaulHancock** [comment]
Would the following be a more appropriate way to compute the number of trials:
```
max_trials = min(NPIX,len(tab))
idx = np.random.choice(range(max_trials), max_trials, replace=False)
```
This way we only ever compute up to NPIX light curves, and if NPIX is more than the number of entries in the table we just use the entire table.

**Comment by JunTian-CIRA** [comment]
I think that should work, thanks.

**Comment by PaulHancock** [comment]
@JunTian-CIRA Can you please test that this new version of the code works (once i've updated the container on Pawsey)?

**Comment by JunTian-CIRA** [comment]
Sure, that's great!

**Comment by PaulHancock** [comment]
@JunTian-CIRA Our discussion yesterday made it sound like this problem has been fixed. If so can you close this issue, or if not, can you update this ticket?

**Comment by PaulHancock** [comment]
Should be fixed as of 0cde5c1
